src/patterns/task-list-pages/index.md.njk

```diff
@@ -149,13 +156,6 @@ Register as a childminder
 
 ### Next steps
 
-Work is needed to ensure that users can understand the difference between completing a task and completing the whole service.
-
-In addition, more user research is needed to find out:
-
-- how to help screen reader users to get an overview of the progress they have made through the task list
-- whether to return users to the task list after each task or take them straight to the next task in the sequence
-- the best way to show when tasks must be completed in a fixed order
-- how to ensure users can see which tasks have been completed and which they still need to do
+A cross-government team are collaborating on work to update this pattern, and to introduce a component.
 
-If you’ve used this component, get in touch to share your user research findings.
+If you’ve used this pattern, [get in touch](/get-in-touch/) to share your user research findings.
```

Comment:
Should the backlog be the place to share this?

Reply:
Yeah, probably the backlog. This link duplicates the 'generic' below but can't think of a clear way to avoid at the mo.

```suggestion
If you’ve used this pattern, [share your user research findings with us on GitHub](https://github.com/alphagov/govuk-design-system-backlog/issues/72) to help us improve it.
```